src/VoteProxy.sol

```diff
@@ -18,18 +18,32 @@
 // vote w/ a hot or cold wallet using a proxy identity
 pragma solidity >=0.4.24;
 
-import "ds-token/token.sol";
-import "ds-chief/chief.sol";
+interface TokenLike {
+    function balanceOf(address) external view returns (uint256);
+    function approve(address, uint256) external;
+    function pull(address, uint256) external;
+    function push(address, uint256) external;
+}
+
+interface ChiefLike {
+    function GOV() external view returns (TokenLike);
+    function IOU() external view returns (TokenLike);
+    function deposits(address) external view returns (uint256);
+    function lock(uint256) external;
+    function free(uint256) external;
+    function vote(address[] calldata) external returns (bytes32);
+    function vote(bytes32) external;
+}
```

Comment:
Alternative here would be to pull in `dss-interfaces`.

Reply:
So far we have been using simpler interfaces (with the minimum) for the actual contracts. But yeah this is probably something to discuss more deeply.